_config.yml

```diff
@@ -8,7 +8,7 @@ markdown: redcarpet
 redcarpet:
   extensions: []
 
-highlighter: pygments
+highlighter: pigments
```

Comment:
It must have changed in newer versions because I had to change it to `pigments` in order to get the project building